phpcs.xml.dist

```diff
@@ -5,8 +5,11 @@
 	<rule ref="PHPCompatibility"/>
 	<config name="testVersion" value="5.6-"/>
 
-	<rule ref="WordPress-Core"/>
 	<rule ref="WordPress-Docs"/>
+	<rule ref="WordPress-Extra">
+		<exclude-pattern>tests/*</exclude-pattern>
+		<exclude-pattern>modules/database/sqlite/*</exclude-pattern>
```

Comment:
Why is this module specifically excluded? Can you please add a code comment to explain, as this isn't exactly clear to me?

Reply:
In https://github.com/WordPress/performance/pull/695/commits/071ffacf0181bf7c3370d3d11a666e5a1ec54a87, I have added `TODO` comment for sqlite exclusion rule.